### PR TITLE
[WIP] Add python SDK generation to submodule

### PR DESCRIPTION
**Pull request by mjudeikis**
Add generation for python SDK into git submodules

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,22 +6,40 @@ rp:
...
 
-client:
+client: client-go client-python
+
+client-go:
 	go generate ./...
 	rm -rf pkg/client
 	sha256sum swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2019-12-31-preview/redhatopenshift.json >.sha256sum
 	sudo docker run \
+		--net host \
```
Comment (mjudeikis) [inline Makefile]: this is breaking my docker builds... might be community version of docker or smth.

Comment (jim-minter) [reply Makefile]: that sounds like a problem to fix on your box, not in this PR?

File: Makefile [hunk 2]
```diff
@@ -6,11 +6,14 @@ rp:
 clean:
 	rm -f rp
 
-client:
+client: client-go client-python
```
Comment (jim-minter) [inline Makefile]: keep all this in one target so you can make use of the sha256sum

File: Makefile [hunk 3]
```diff
@@ -22,6 +25,21 @@ client:
...
 	go run ./vendor/golang.org/x/tools/cmd/goimports -w -local=github.com/jim-minter/rp pkg/client
 
+client-python:
+	go generate ./...
+	sudo docker run \
+		--net host \
+		-v $(PWD)/az:/az \
+		-v $(PWD)/swagger:/swagger \
+		azuresdk/autorest \
+		--use=@microsoft.azure/autorest.python@4.0.70 \
```
Comment (jim-minter) [inline Makefile:28]: what is special about this version?

Comment (jim-minter) [reply Makefile:28]: answer: it's well hidden, but it bumps the version used to latest

**Comment by jim-minter** [comment]
@mjudeikis added some changes but hopefully good - merging

**Pull request merged**